assembl/static2/js/app/components/debate/common/replyToCommentButton.jsx

```diff
@@ -1,16 +1,22 @@
 // @flow
 import React from 'react';
 import { Link } from 'react-router';
+import classNames from 'classnames';
 
 export type ReplyToCommentButtonProps = {
   /** On click callback function */
-  onClickCallback: Function
+  onClickCallback: Function,
+  disabled?: boolean
 };
 
-const ReplyToCommentButton = ({ onClickCallback }: ReplyToCommentButtonProps) => (
-  <Link className="action-reply" onClick={onClickCallback}>
+const ReplyToCommentButton = ({ onClickCallback, disabled }: ReplyToCommentButtonProps) => (
+  <Link className={classNames('action-reply', { disabled: disabled })} onClick={!disabled ? onClickCallback : null}>
```

Comment:
about the onClick: isn't the css disabling of the button enough ?

Reply:
@MilanBarande @pascalhuynh If it was a button, I could use the 'disabled' selector but here it's a link so I have to cancel the callback and add a css class to change color

Reply:
It s a bootstrap feature that doesn't work with links like this, without href but this works in css: pointer-events: none;
  cursor: default;